src/Common/Exception/NotFoundException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Omnipay\Common\Exception;
+
+/**
+ * Not Found Exception
+ *
+ * Thrown when the requested resource is not found on payment gateway
+ */
+class NotFoundException extends \Exception implements OmnipayException
+{
+    public function __construct($message = "Resource not found on payment gateway", $code = 404, $previous = null)
```

Comment:
I think the code can be just 0

Reply:
Do you want me to change or 404 is also OK?

Reply:
Changed to 0

Reply:
Change to 0